provider/datagen/src/cldr/transform/datetime/mod.rs

```diff
@@ -44,25 +46,21 @@ macro_rules! impl_resource_provider {
                     &self,
                     req: &DataRequest,
                 ) -> Result<DataResponse<$marker>, DataError> {
-                    let langid = req
-                        .get_langid()
-                        .ok_or_else(|| DataErrorKind::NeedsLocale.with_req(<$marker>::KEY, req))?;
-                    let variant = req
-                        .options
-                        .variant
-                        .as_ref()
-                        .ok_or_else(|| DataErrorKind::NeedsVariant.with_req(<$marker>::KEY, req))?;
+                    let langid = req.langid();
```

Comment:
I interpret this as a `Suggestion`. I'll remove the extra method.

Reply:
personally i think this one is okay, i think it would be pretty common to want the langid and it's not _that_ different in ergonomics but it's a good signal in the docs that "hey, this is how you get the langid" where the two-level method doesn't have one

Reply:
I think it's good to remove the method since we have a more robust set of methods on the inner type.